### PR TITLE
[6.5][NET] Changed widget-control to widget-management.

### PR DESCRIPTION
**Pull request by mihashco**
Changes:
- Title was changed because widget-control suggest something that controll
widget, but widget-control is responsible for widget applications
managent like viewing installed widgets list, removing widget etc.
- Some code snippets was fixed.

### CODE REVIEW

**Review by sagarkedari — CHANGES_REQUESTED** [review]
@mihashco , I have reviewed the files and added my feedback comments. Kindly check and incorporate the changes.

**Review by sagarkedari — APPROVED** [review]
@annie-samsung, The PR is approved from my end.

**Comment by annie-samsung** [comment]
@mihashco, @sagarkedari is this specific to 6.5 release? Why is this PR raised to 6.5 branch

File: docs/application/dotnet/guides/app-management/widget-management.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Widget Control
+# Widget Management
```
Comment (sagarkedari) [inline docs/application/dotnet/guides/app-management/widget-management.md:1]: @mihashco, Per below comment, please change the base branch from "Tizen 6.5" to "master" for this PR.
https://github.com/Samsung/tizen-docs/pull/1507#issuecomment-926521854

@annie-samsung, You are correct, this PR should have been raised on master branch. Thank you for pointing this out.

**Review by sagarkedari — CHANGES_REQUESTED** [review]
@mihashco, Please update as per comment provided.

**Comment by mijin85cho** [comment]
Thanks for reviewing & merging this PR @annie-samsung 
However, as Tizen 6.5 is published last week, the updates to tizen_6.5_prepare will not be applied to master.
(tizen_6.5_prepare is merged to master last week.)

So for the PR to tizen_6.5_prepare, 
please guide authors to change the target branch to master, or create a new PR that works on master branch.

**Comment by annie-samsung** [comment]
@mihashco, could you please raise this PR to master branch, could you do that for all your PRs?

**Comment by annie-samsung** [comment]
> Thanks for reviewing & merging this PR @annie-samsung However, as Tizen 6.5 is published last week, the updates to tizen_6.5_prepare will not be applied to master. (tizen_6.5_prepare is merged to master last week.)
> 
> So for the PR to tizen_6.5_prepare, please guide authors to change the target branch to master, or create a new PR that works on master branch.

@mijin85cho, I had mentioned this in chat, to Michal,  I was in two minds should I close or merge this PR post approving.

**Pull request merged**